Run7/step14/dark_shutterClosed.cfg

```diff
@@ -0,0 +1,8 @@
+#  20 15sec DARK images taken with room lights off and Shutter CLOSED
+# Before executing this config, make sure shutter is closed. The SHUTTER=None keyword
+[ACQUIRE]
+dark
+[DARK]
+BCOUNT=0
+dark=15.0 20     # integration time and image count for dark set
+SHUTTER=None
```

Comment:
What does `SHUTTER=None` mean?  I don't see it used anywhere else in this repo.  I do see
```
SHUTTER  = CLOSE
```
in `Run6/step13/opsimdark.cfg`.   Based that config, I would have expected the entries here to be
```
[DARK]
SHUTTER = CLOSE
BCOUNT=0
dark=15.0 20     # integration time and image count for dark set
```
with the `SHUTTER = CLOSE` entry before the `dark=15.0 20` entry.

Reply:
[Explanation on the shutter entries is here](https://github.com/lsst-camera-dh/fp-scripts/blob/b07be5358a0afadd713b1f13b063914f897973ac/lib/acquire.py#L49-L52).

There is no `SHUTTER = CLOSE` option supported. That is why I specify the comment at the head of the config file

Reply:
Here is your comment:
```
 Before executing this config, make sure shutter is closed. The SHUTTER=None keyword
```
I don't know what "The SHUTTER=None keyword" is supposed to mean.  That's why I asked.